com.musixmatch.Musixmatch.desktop

```
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Musixmatch
+Comment=Musixmatch Desktop App
+Exec=musixmatch %U
+Terminal=false
+Type=Application
+Icon=com.musixmatch.Musixmatch
+StartupWMClass=Musixmatch
+MimeType=application/mxm;x-scheme-handler/mxm;
+Categories=Utility;
```

I would add Audio as a category here.
```suggestion
Categories=Utility;Audio;
```